README.md

```diff
@@ -59,6 +59,28 @@ from other services too: you'll just need to provide your file URLs for import.
                                   check requests.
                                   
   ```              
+ 
+## Output file format
+The output file format is very simple.
+For every file URL or Filestack file handle there is 3 columns divided by '\t'(tab).
+
+First column - URL of Filestack handle iteself. 
+
+Second column - upload operation status - `succes` or `failed`.
+
+Third column - The Uploadcare URL of uploaded file or error description.
+
+For instance, if you have 3 files to migrate to Uploadcare and your input file looks like this:
+    
```

Comment:
For instance, you're willing to migrate the three following files to Uploadcare where the first two are presented as URL and the third one as Filestack file handle. Also, that's how your input text file will then be structured: